test/role.test.js

```diff
@@ -172,6 +173,14 @@ describe('role model', function() {
   });
 
   it('should support owner role resolver', function() {
+    Role.registerResolver(Role.EVERYONE_PROMISE, function(role, context) {
+      process.nextTick(function() {
```

Comment:
This role resolver returns `undefined`, which is probably not what you wanted. The trick is to create the promise first:

```js
return new Promise(function(resolve, reject) {
  process.nextTick(resolve);
}
```